.gitignore

```diff
@@ -37,6 +37,7 @@ _testmain.go
 
 *.iml
 .idea/
+.run
```

Comment:
This is not generated by the kube-state-metrics repository so I wouldn't want to add it to the repository. If this is annoying in your environment I would recommend using a global gitignore.